bin/utils/subkey/secret

```diff
@@ -0,0 +1 @@
+�.o��������R�c�3J}��8���b�}&Ai
```

Comment:
remove it?

Reply:
heh, yes accidentally committed while testing on another branch - deleted